frontend/src/components/VideoCarrouselSlick.js

```diff
@@ -8,7 +8,6 @@ import '../style/style.scss';
 
 class VideoCarrouselSlick extends Component {
 
-    CARROUSSEL_SIZE = 5;
     VIDEO_PER_PAGE = 10;
```

Comment:
We should avoid using this hardcoded param here. What if we change the number of video per page in the backend and forget to change it here?

So I guess we could use the number of video retrived in the last page for the caculations done later. 
See my next comment.